### ISSUE TITLE
Malfunction after exiting sleep mode with Ledger Live running

### ISSUE THREAD

**Issue by msvprogs** [LedgerHQ/ledger-live-desktop#1516]
When I exit the sleep mode with running Ledger Live app, its window becomes empty and doesn't recover until I rerun the program.
![default](https://user-images.githubusercontent.com/28513129/45598584-402bdb00-b9e7-11e8-985b-ddfeb04edda4.png)

#### Ledger Live Version and Operating System

- tested on Ledger Live 1.1.10
- Platform and version: Windows 10 Pro

#### Expected behavior
After exiting the sleep mode Ledger Live displays the block screen with passwordbox or the screen it was displaying before entering sleep mode. 

#### Actual behavior

After exiting the sleep mode Ledger Live shows an empty window and stops working.

#### Steps to reproduce the behavior

1. Run Ledger Live app. 
2. Enter sleep mode. 
3. Exit from it. 
4. Ledger Live shows an empty window.

**Comment by gre** [comment]
I would really like to understand how this can happen. do you have any logs? (try to Ctrl+E when this happen – send the log files to our support team, not on github)

**Comment by msvprogs** [comment]
Ctrl+E had no effect on hanged application :(

**Comment by gre** [comment]
I currently suspect a filesystem issue, potentially a permission issue on the App Data app.json file writing/reading. Have you ever run the app in Administrator? (i wonder if writing the app.json in admin mode would trigger an issue if you later open as a normal user).

I'm not sure what's the ways to check the permission of files on Windows, but you might try to investigate what's going on in this App Data folder. (located under `%APPDATA%\Ledger Live`)

Secondly, can you attempt to run the app in Administrator see if it helps at all to not have the white screen.

I really hope we can tackle this down, I didn't had yet the change to reproduce the bug.
thanks.

**Comment by msvprogs** [comment]
No, running the app under admin rights didn't help, it still displays white screen.
I haven't run it under admin before, so there shouldn't be any problem with file permissions, especially in the user's home directory.
However, I discovered the log file there and found out that some kind of error does happen after waking up:
```
{"type":"db","level":"debug","message":"📁  read app:user","pname":"main","timestamp":"2018-09-17T17:27:27.943Z"}
{"type":"db","level":"debug","message":"📁  read app:MainWindow.dimensions","pname":"main","timestamp":"2018-09-17T17:27:27.946Z"}
{"type":"db","level":"debug","message":"📁  read app:MainWindow.positions","pname":"main","timestamp":"2018-09-17T17:27:27.962Z"}
{"type":"db","level":"debug","message":"📁  write windowParams:MainWindow.positions","pname":"main","timestamp":"2018-09-17T17:27:28.584Z"}
{"level":"info","message":"booting internal process...","pname":"main","timestamp":"2018-09-17T17:27:31.836Z"}

// ***** these lines appear AFTER exiting the sleep mode *****
{"type":"db","level":"debug","message":"📁  write windowParams:MainWindow.positions","pname":"main","timestamp":"2018-09-17T17:30:04.259Z"}
{"level":"warn","message":"Internal process ended with code 3221226525","pname":"main","timestamp":"2018-09-17T17:30:24.751Z"}
```

**Comment by teams2ua** [comment]
Have the same issue, the utility processes crashed after  wake up. The time I have run this on developer environment the exception was on node usb module.

**Comment by gre** [comment]
We expect this to be fixed in context of a big rework that just got merged. we'll reopen in this does not fix in the next release #2596
